models/userDao.js

```diff
@@ -0,0 +1,78 @@
+const { appDataSource } = require("./dataSource");
+
+const createUser = async (
+  name,
+  email,
+  phone_number,
+  birthday,
+  gender,
+  address,
+  address_detail,
+  password,
+  point,
+  agreement_private,
+  agreement_marketing,
+  agreement_terms
+) => {
+  try {
+    return await appDataSource.query(
+      `INSERT INTO users(
+                name,
+                email,
+                phone_number,
+                birthday,
+                gender,
+                address,
+                address_detail,
+                password,
+                point,
+                agreement_private,
+                agreement_marketing,
+                agreement_terms
+		) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
+		`,
+      [
+        name,
+        email,
+        phone_number,
+        birthday,
+        gender,
+        address,
+        address_detail,
+        password,
+        point,
+        agreement_private,
+        agreement_marketing,
+        agreement_terms,
+      ]
+    );
+  } catch (err) {
+    const error = new Error("INVALID_DATA_INPUT");
+    error.statusCode = 500;
```

Comment:
상태코드 500은 에러처리를 안 했다는 의미가 될 수 있기 때문에 400번대 상태코드를 사용해 주세요.
```suggestion
    error.statusCode = 400;
```